docs/buildah-unshare.md

```diff
@@ -0,0 +1,32 @@
+# buildah-unshare "19" "June 2018" "buildah"
+
+## NAME
```

Comment:
Buildah is all about containers and images, but not a mention of either in this man page.  Can you tie in how this command would be useful to either or both in this page?

Reply:
Updated to note that this is mainly for troubleshooting and manual clearing of storage areas, and updated the example that deletes everything to reflect the updated locations.

Reply:
ty!